src/crypto/crypto.gni

```diff
@@ -15,6 +16,7 @@
 declare_args() {
   # Crypto implementation: mbedtls, openssl.
   chip_crypto = ""
+  with_se05x = 0
```

Comment:
I think we standardise to using `chip_` as a prefix for all gn arguments added by chip.

Reply:
Sure it is now chip_with_se05x